app/Http/Controllers/Api/V1/TeamController.php

```diff
@@ -675,7 +699,10 @@ public function update(TeamUpdateRequest $teamUpdateRequest, Organization $subor
         //     $exist_user_id = $suborganization->users()->where('user_id', $user['id'])->first();
         //     if (!$exist_user_id) {
         //         return response([
-        //             'errors' => ['Team not created, ' . $user['email'] . ' must be added in ' . $suborganization->name . ' organization first.'],
+        //             'errors' =>
```

Comment:
Please revert it back to previous, if it makes horizontal scroll then only we use break line